WindowManager.cs

```diff
@@ -549,9 +569,11 @@ public override void Update(GameTime gameTime)
             XNAControl activeControl = null;
             activeControlName = null;
 
+#if WINFORMS
             if (HasFocus)
                 Keyboard.Update(gameTime);
 
+#endif
```

Comment:
As far as I can see, the above causes the Keyboard not to be updated unless the WinForms build is used.

Restoring the `HasFocus` property would fix this.